api/src/main/java/org/jboss/el/cache/BeanPropertiesCache.java

```diff
@@ -177,49 +203,45 @@ public BeanProperty getBeanProperty(String property) {
 
 
     /*
-     * Get a public method form a public class or interface of a given method.
-     * Note that if a PropertyDescriptor is obtained for a non-public class that
-     * implements a public interface, the read/write methods will be for the
-     * class, and therefore inaccessible.  To correct this, a version of the
-     * same method must be found in a superclass or interface.
+     * This is the same as the ELUtil.getMethod implementation in the Eclipse Jakarta EL project.
+     * That class is package protected so we can't call it from this class.
+     * In the org.jboss fork, the ELUtil method delegates to this. This is to avoid having
+     * two impls of the same code and
```

Comment:
Should the `same code and` comment be followed by more or the `and` removed?

Reply:
Thanks. I removed the 'and'.